articles/quickstart/native/react-native/_includes/_login.md

```diff
@@ -0,0 +1,99 @@
+The first step in adding authentication to your application is to provide a way for your users to log in. The fastest, most secure, and most feature-rich way to do this with Auth0 is to use the [login page](https://auth0.com/docs/hosted-pages/login).
+
+<div class="phone-mockup"><img src="/media/articles/native-platforms/ios-swift/lock_centralized_login.png" alt="Hosted Login Page"></div>
+
+## Native configuration
+
+Auth0 will need to handle the callback of the hosted login authentication.
+
+### Android
+
+In the file `android/app/src/main/AndroidManifest.xml` you must make sure the main activity of the app has **launchMode** value of `singleTask` and that it has the following intent filter.
+
+```xml
+<activity
+android:name=".MainActivity"
+android:label="@string/app_name"
+android:launchMode="singleTask"
+android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
+android:windowSoftInputMode="adjustResize">
+<intent-filter>
+    <action android:name="android.intent.action.MAIN" />
+    <category android:name="android.intent.category.LAUNCHER" />
+</intent-filter>
+<intent-filter>
+    <action android:name="android.intent.action.VIEW" />
+    <category android:name="android.intent.category.DEFAULT" />
+    <category android:name="android.intent.category.BROWSABLE" />
+    <data
+        android:host="${account.namespace}"
+        android:pathPrefix="/android/<%= "${applicationId}" %>/callback"
+        android:scheme="<%= "${applicationId}" %>" />
+</intent-filter>
+</activity>
+```
+
+Replace `YOUR_APP_PACKAGE_NAME` with your actual application's package name.
+
+### iOS
+
+In the file `ios/<YOUR PROJECT>/AppDelegate.m` add the following:
```

Comment:
Because in Android it is literally `/00-Login/android/app` regardless of project name
In iOS it is `/00-Login/ios/Auth0Sample` with the actual project name as part of the path.